sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3063,6 +3063,20 @@ def set_autostop(self,
             global_user_state.set_cluster_autostop_value(
                 handle.cluster_name, idle_minutes_to_autostop, down)
 
+    def is_autostopping(self,
+                        handle: ResourceHandle,
+                        stream_logs: bool = True) -> bool:
+        code = autostop_lib.AutostopCodeGen.is_autostopping()
+        returncode, stdout, stderr = self.run_on_head(handle,
+                                                      code,
+                                                      require_outputs=True,
+                                                      stream_logs=stream_logs)
+
+        if returncode == 0:
+            return common_utils.decode_payload(stdout)
+        logger.debug(f'Failed to check if cluster is autostopping: {stderr}')
+        return False
```

Comment:
More like `is_definitely_autostopping()` due to this. Not feeling strongly, however. Maybe add docstr saying True = definitely, False = maybe?